requirements.txt

```diff
@@ -1,7 +1,7 @@
 matplotlib==3.7.2
-numpy==1.26.1
+numpy==1.24.4
 pandas==1.3.5
-scipy==1.11.2
+scipy==1.10.1
```

Comment:
@cvetkovic We noticed that this bump of numpy and scipy versions make it impossible to use sampler on Ubuntu 20.04 (I think that's the reason, but might be other factor). Do you know a proper way of fixing that?

Reply:
It is due to the Python version, but still, the Python version is linked to the Ubuntu version which is annoying, so I lowered the dependency versions. The numpy 1.26.1 and scipy 1.10.1 require Python 3.9, and we don't have that as default on Ubuntu 20.04

Reply:
ok for now but we need to move to using ubuntu 22 for measurements. this is about time but we need to plan this upgrade vHive-wide.